tests/tracking/test_tracking.py

```diff
@@ -185,9 +189,8 @@ def test_log_batch(tracking_uri_mock):
 
 
 def test_log_metric(tracking_uri_mock):
-    active_run = start_run()
-    run_uuid = active_run.info.run_uuid
-    with active_run:
+    with start_run() as active_run:
```

Comment:
NOTE: code changes starting here are related only to `with` block refactoring, with one exception (see comment below)